usersync/usersyncers/syncer.go

```diff
@@ -148,6 +149,7 @@ func NewSyncerMap(cfg *config.Configuration) map[openrtb_ext.BidderName]usersync
 	insertIntoMap(cfg, syncers, openrtb_ext.BidderYieldmo, yieldmo.NewYieldmoSyncer)
 	insertIntoMap(cfg, syncers, openrtb_ext.BidderYieldone, yieldone.NewYieldoneSyncer)
 	insertIntoMap(cfg, syncers, openrtb_ext.BidderZeroClickFraud, zeroclickfraud.NewZeroClickFraudSyncer)
+	insertIntoMap(cfg, syncers, openrtb_ext.BidderBetween, between.NewBetweenSyncer)
```

Comment:
Since your adapter doesn't support usersyncing, you don't need this

Reply:
I'll delete all these usersync leftovers for now, and return later, after we done with usersync testing internally, thanks for pointing this out.

Reply:
Sounds great! Thanks @digitalist